templates/_partials/fields/select.html.twig

```diff
@@ -15,6 +15,12 @@
     {% set multiple = 'false' %}
 {% endif %}
 
+{% if allowempty is not defined and field.definition is defined %}
```

Comment:
is it possible to have field without it's definition?

Reply:
Currently, yes. The language switcher, and the `<select>` for taxonomies, as well as the forms on the "profile page" and the "edit media" page use the same field twig components, only they aren't tied to a Field, so they don't have a definition per se.

Reply:
Ok, so this can only check `field is defined` instead of `field.definition is defined` (because it will probably break if `field` is not set at all).

This check should be probably moved to `_base.html.twig` as mentioned in https://github.com/bolt/four/issues/294#issuecomment-457337734 or, if `allowempty` is something special for select field, document it with comment on top of this template.

Reply:
The `allowempty` check is specific for select fields, so i've kept it here, but I've added a comment describing it. 👍

Reply:
It will still break if field is not defined